react-app-rewired/headers/csps/defi/safe.ts

```diff
@@ -2,6 +2,7 @@ import type { Csp } from '../../types'
 
 export const csp: Csp = {
   'connect-src': [
+    'https://app.safe.global',
```

Comment:
Also in preparation for ShapeShift as a SAFE app, which this PR will now allow with its EIP-6963 detection